helios-tools/src/main/java/com/spotify/helios/cli/command/RollingUpdateCommand.java

```diff
@@ -131,6 +132,15 @@ public Long get() {
         .setDefault(EMPTY_TOKEN)
         .help("Insecure access token meant to prevent accidental changes to your job "
               + "(e.g. undeploys).");
+
+    ignoreFailuresArg = parser.addArgument("--ignore-failures")
+        .setDefault(false)
+        .action(storeTrue())
+        .help("When specified, the rolling update will continue on to all hosts in the deployment "
+              + "group without waiting for the job to reach the RUNNING state on each host. "
+              + "Be *very* careful about using this option, as it has the potential to completely "
+              + "take down your service by rolling out a broken job to all of the hosts in your "
```

Comment:
Perhaps mention that this also effectively makes the rollout "fully parallel"? Specifying `--ignore--failures` will, aside from ignoring errors, also cause similar behaviours as when using e.g. `--par 10000`